src/main/scala/textEvidence/TextEvidenceExtractor.scala

```diff
@@ -72,22 +84,13 @@ class TextEvidenceExtractor extends JenaFragmentsWrapper{
     tokenized.map(token => PorterStemmer(token)).toList
   }
 
-  def getAbstractsAndTitle(uris: List[String]) : Future[List[(List[String], List[String])]] = {
-    /*
-    for {
-      uri <- uris
-      titleText <- getTitle(uri)
-      abstractText <- getAbstract(uri)
-    } yield (titleText, abstractText)
-    */
-
-    val abstractsMap = for (uri <- uris) yield async {
+  def getAbstractsAndTitle(uri: String): Future[(List[String], List[String])] = {
+    async {
       val titleText = await(getTitle(uri))
       val abstractText = await(getAbstract(uri))
 
```

Comment:
yes but it wasnt possible to load the two dumps into one database. So I splitted them again. But since this are no HTTP request but file reads it could even be faster this way